### PR TITLE
Refactor: Apply `AspectRatio` to iframe in "Attack and Defense" page

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description

Adds Chakra's `AspectRatio` component to wrap the embedded iframe used in the PoS [Attack and Defense](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/attack-and-defense/#reorgs) page (under the "reorgs" section).

## Related Issue
N/A

### CODE REVIEW

**Review by minimalsm — COMMENTED** [review]
LGTM :-) 

Curious if we should use [the YouTube version of this video](https://www.youtube.com/watch?v=xcPxwhrg3Ao) instead? Or did you deliberately want to avoid YouTube @jmcook1186 here?

**Comment by TylerAPfledderer** [comment]
Hey @minimalsm ! 😀

That component is rendering YouTube-specific attributes values, such as generating a Youtube url string with a passed-in user id value.

**Comment by TylerAPfledderer** [comment]
Hey @minimalsm ! 😄

I didn't know one existed! I'll take a look at that.

**Comment by TylerAPfledderer** [comment]
@minimalsm ok, I have updated the embed to Youtube!

File: src/@chakra-ui/gatsby-plugin/styles.ts [hunk 1]
```diff
@@ -58,12 +58,6 @@ const styles = {
     "li > p": {
       "margin-bottom": "calc(1.45rem / 2)",
     },
-    // YouTube embeds
-    iframe: {
-      display: "block",
-      maxWidth: "560px",
-      margin: "32px 0",
-    },
```
Comment (pettinarip) [inline src/@chakra-ui/gatsby-plugin/styles.ts:66]: 🙏🏼

**Review by pettinarip — APPROVED** [review]
Yey!! 🎉 YT centralization 🤪

**Pull request merged**